connectors/sources/jira.py

```diff
@@ -320,8 +371,310 @@ def get_default_configuration(cls):
+                            if self._is_active_atlassian_user(user_info=user):
+                                access_control.add(
+                                    _prefix_account_id(user_id=user.get("accountId"))
+                                )
+                                access_control.add(
+                                    _prefix_username(user=user.get("displayName"))
+                                )
+                        elif actor.get("holder", {}).get("type") == "group":
+                            group_id = (
+                                actor.get("holder", {}).get("group", {}).get("groupId")
+                            )
+                            access_control.add(_prefix_group_id(group_id=group_id))
+                        elif actor.get("holder", {}).get("type") == "projectRole":
+                            if (
+                                role_id := actor.get("holder", {})
+                                .get("projectRole", {})
+                                .get("id")
+                            ):
+                                # Project Role - `atlassian-addons-project-access` with id 10003 is not needed for DLS
+                                if role_id == 10003:
+                                    continue
+                                access_control = await anext(
+                                    self._project_role_members(
+                                        project=project,
+                                        role_id=role_id,
+                                        access_control=access_control,
+                                    )
+                                )
+            else:
+                self._logger.debug(
+                    f"Issue security level is not set for an issue: {issue_key}. Hence, Assigning project permissions"
+                )
+                project_access_controls = await self._project_access_control(
+                    project=project
+                )
+                return project_access_controls
+        return list(access_control)
+
+    async def _user_access_control_doc(self, user):
+        """Constructs a user access control document, which will be synced to the corresponding access control index.
+        The `_id` of the user access control document will either be the username (can also be the email sometimes) or the email itself.
+        Note: the `_id` field won't be prefixed with the corresponding identity prefix ("user" or "email").
+        The document contains all groups of a user and his email and/or username under `query.template.params.access_control`.
+
+        Returns:
+            dict: dictionary representing an user access control document
+            {
+                "_id": "some.user@spo.com",
+                "identity": {
+                    "email": "email:some.user@spo.com",
+                    "username": "user:some.user",
+                    "user_id": "user_id:some user id"
```

Comment:
Here you're using `email`, `username` and `user_id` as the fields for `identity`, but I only see `account_id` and `display_name` down below. Should we adjust that?

Reply:
Yes, this should be adjusted.